### ISSUE TITLE
Make it easier to build the docs

### ISSUE THREAD

**Issue by rgommers** [mesonbuild/meson-python#368]
_Note: not high-prio, please leave this until after we get the 0.13.0 release out. Just documenting now because I'm running into it during review of another PR._

Two issues:

1. The expected command for sphinx is `make html`, so let's add the Makefile that Sphinx generates by default. It's low-maintenance, it basically never changes (all it does is automate the much longer `sphinx-build .....` invocation).
2. Along the way I lost the ability to build the docs on Linux (see below). Let's make that easier by getting rid of spell checking, that kind of thing really should be a CI-only check.

Doc build issue:
```
Exception occurred:
  File "/home/rgommers/code/meson-python/.nox/docs/lib/python3.10/site-packages/enchant/__init__.py", line 233, in _raise_error
    raise eclass(default)
enchant.errors.DictNotFoundError: Dictionary for language 'en_US' could not be found
Please check https://pyenchant.github.io/pyenchant/ for details
The full traceback has been saved in /tmp/sphinx-err-x5c1yxb3.log, if you want to report the issue to the developers.
```

Uninstalling/reinstalling `pyenchant` didn't help, installing `enchant` and `aspell-en` with `pacman` didn't help. This wastes too much time, `sphinxcontrib-spelling` is not a good dependency since it hard-depends on non-PyPI stuff.

**Comment by rgommers** [comment]
> there are many TODO markers in the documentation that need to be addressed before we can release 0.13.0

**Comment by rgommers** [comment]
> there are many TODO markers in the documentation that need to be addressed before we can release 0.13.0 

I don't really see those as blockers, since they're indeed already up on https://meson-python.readthedocs.io now. Where we're at now is:
- we have a major new feature in `main`, editable installs, that seems to be working quite well and lots of users are asking for.
- we also have a number of useful bug fixes in `main` that folks are now having to work around (I [ported PyWavelets](https://github.com/PyWavelets/pywt/pull/667) over the weekend, and noticed I had to copy some workarounds from elsewhere for example)
- I wanted to see gh-348 finalized (now gh-364 too) before the release, because it seemed very close already two weeks ago, just some RPATH tweaks away.
- the state of `main` is a major improvement over `0.12.0` either way, and I don't think there are regressions or hard blockers.

I agree that we need to work on the TODOs in the docs, but what we really need asap is to get editable installs shipped - everything else is secondary to that.

**Comment by rgommers** [comment]
Fixed by gh-408, so closing.
